### PR TITLE
Changed .htaccess for OWLChangeOntology

### PR DESCRIPTION
**Pull request by DiegoCondeHerreros**

### CODE REVIEW

**Comment by dgarijo** [comment]
Have you tried this locally, @DiegoCondeHerreros?
you are removing the application types, which are used to indicate the type of resource being returned in the response. 
You are also adding `^(.*)\.resource.conneg` which does not make a lot of sense to me

File: OWLChangeOntology/.htaccess [hunk 1]
```diff
@@ -1,39 +1,20 @@
...
+RewriteCond %{HTTP_ACCEPT} text/html
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteRule ^(.*)\.resource.conneg$ https://diegocondeherreros.github.io/OWLChangeOntology/index.html [NE,R,L]
 RewriteCond %{HTTP_ACCEPT} .+
-RewriteRule ^$ 406.html [R=406,L]
\ No newline at end of file
+
+RewriteRule ^/$ https://%{SERVER_NAME}/OWLChangeontology/ontology.resource.conneg [NE,R,L]
+# Rewrite engine setup
+RewriteEngine On
```
Comment (davidlehn) [inline OWLChangeOntology/.htaccess:20]: This is usually nearbefore the rewrite rules.  Maybe required to be, I'm not sure.

File: OWLChangeOntology/.htaccess [hunk 2]
```diff
@@ -1,39 +1,20 @@
...
+RewriteCond %{HTTP_ACCEPT} text/turtle
+RewriteRule ^(.*)\.resource.conneg$ https://diegocondeherreros.github.io/OWLChangeOntology/$1.ttl [NE,R,L]
+RewriteCond %{HTTP_ACCEPT} text/html
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteRule ^(.*)\.resource.conneg$ https://diegocondeherreros.github.io/OWLChangeOntology/index.html [NE,R,L]
 RewriteCond %{HTTP_ACCEPT} .+
-RewriteRule ^$ 406.html [R=406,L]
\ No newline at end of file
+
+RewriteRule ^/$ https://%{SERVER_NAME}/OWLChangeontology/ontology.resource.conneg [NE,R,L]
```
Comment (davidlehn) [inline OWLChangeOntology/.htaccess:18]: This pattern is odd and I think will do a secondary user agent request.  Could probably be optimized to not do that.  Also it has a path typo.

**Comment by dgarijo** [comment]
ping @DiegoCondeHerreros

**Pull request closed without merging**